config.txt

```diff
@@ -0,0 +1,29 @@
+Landorus-incarnate
```

Comment:
that file should be deleted from the depot

Reply:
deleted!